data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_abilities.lst

```diff
@@ -27,11 +27,11 @@ Afflicted Kender (Krynn)						KEY:Kender ~ Afflicted Kender (Krynn)	CATEGORY:Int
 
 
 # Standard Kender
-+1 Racial bonus								KEY:Kender ~ Save Bonus				CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																															BONUS:SAVE|Fortitude,Reflex,Will|1|TYPE=Racial
++1 Racial bonus to save								KEY:Kender ~ Save Bonus				CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																															BONUS:SAVE|Fortitude,Reflex,Will|1|TYPE=Racial
 +2 Racial bonus to spot							KEY:Kender ~ Spot Bonus				CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																						BONUS:SKILL|Spot|2|TYPE=Racial
 +2 Racial bonus to Open Locks and Sleight of Hand		KEY:Kender ~ Thievery Bonus			CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																						BONUS:SKILL|Open Lock,Sleight of Hand|2|TYPE=Racial
 Lack of Focus								KEY:Kender ~ Lack of Focus			CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																						BONUS:SKILL|Concentration|-4
-Taunt										KEY:Kender ~ Taunt				CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait
+Taunt +4 bonus to bluff										KEY:Kender ~ Taunt				CATEGORY:Special Ability	TYPE:SpecialQuality.Kender Race Trait																																																																		BONUS:SKILL|Bluff|4|TYPE=Racial
```

Comment:
The "+4 bonus to bluff" should be its own aspect tag. ASPECT:SaveBonus|+4 Bonus to bluff